hadoop-ozone/dev-support/checks/_lib.sh

```diff
@@ -72,3 +72,63 @@ create_aws_dir() {
     popd
   fi
 }
+
+
+# Function to find project paths for a given test class
+# Returns an array of project paths in PROJECT_PATHS
+find_project_paths_for_test_class() {
```

Comment:
Changes in `_lib.sh` trigger full CI run.  I would prefer to add this function as a helper script in `dev-support/ci`, and add exclusions in `dev-support/ci/selective_ci_checks.sh`, similar to `pr_title_check`.